src/instance.ts

```diff
@@ -392,7 +392,7 @@ function setupWatchRun(compiler, instanceName: string) {
         instance.watchedFiles = set;
 
         const instanceTimes = instance.times;
-        instance.times = Object.assign({}, times) as any;
+        instance.times = { ...times } as any;
```

Comment:
This is the only case that uses `Object.assign()` (i.e. `object-assign` module).